osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/alarm/ClearAlarmRegisterCommandExecutor.java

```diff
@@ -32,11 +37,17 @@ public class ClearAlarmRegisterCommandExecutor
 
   final DlmsObjectConfigService dlmsObjectConfigService;
 
+  final ObjectConfigService objectConfigService;
+
   private static final int ALARM_CODE = 0;
+  static final int ALARM_REGISTER_ATTRIBUTE_ID = 2;
 
-  public ClearAlarmRegisterCommandExecutor(final DlmsObjectConfigService dlmsObjectConfigService) {
+  public ClearAlarmRegisterCommandExecutor(
+      final DlmsObjectConfigService dlmsObjectConfigService,
```

Comment:
Is this service still in use?

Reply:
No this is not in use anymore and will remove it.